### ISSUE TITLE
Q: Anyone maintaining this repo? Any forks being maintained??

### ISSUE THREAD

**Issue by kkarhan** [RuiSiang/PoW-Shield#611]
Aside from the [demise of the maintainer]( https://www.youtube.com/watch?v=EJAs9Nb-XE8&t=405s ), @RuiSiang, does anyone continue to develop and maintain this Software?

- Cuz to me it does seem like a good solution that doesn't rely on ClownFlare and other value-removing rentseekers acting as middlemen, but instead a working WAF that doesn't mine XMR or shitcoins or require JS...

**Comment by fixedzik** [comment]
he's actually in jail bc he's steal money in darknet. all projects on this acc most probably they will be discontinued.

**Comment by mwangi-brian** [comment]
@kkarhan I don't think so. Especially not when the issues section is filled with braindead crybabies.
